mswindows/osgeo4w/env.bat.tmpl

```diff
@@ -8,6 +8,8 @@ REM Uncomment if you want to use Bash instead of Cmd
 REM Note that msys package must be also installed
 REM set GRASS_SH=%OSGEO4W_ROOT%\apps\msys\bin\sh.exe
 
+set PYTHONPATH=%OSGEO4W_ROOT%\apps\grass\grass@POSTFIX@\etc\python;%PYTHONPATH%
+set GRASS_COMPATIBILITY_TEST=0

```

Comment:
what does this mean?

Reply:
The first one, PYTHONPATH, is needed, I often had to add it to make windows tests work in CI (like when working hard to get pytest running), but the second one I don't know

Reply:
the fatal version check in gisinit.c that used to be patched out in osgeo4w now just causes a warning if GRASS_COMPATIBILITY_CHECK is 0.

Reply:
Oh, so you're talking about this part:
https://github.com/OSGeo/grass/blob/3e22b87722fb96c86c27f3e78405bc8c4fe0ed54/lib/gis/gisinit.c#L43-L73

Shouldn't it be `GRASS_COMPATIBILITY_TEST`?

Reply:
> Shouldn't it be `GRASS_COMPATIBILITY_TEST`?

The comment by Jürgen was edited to match this

Reply:
Thus, this explained what that change was, and this conversation will be resolved